packages/core/entity/FileDetail/index.ts

```diff
@@ -262,7 +262,7 @@ export default class FileDetail {
             if (isFileRenderableImage) {
                 return this.downloadPath;
             }
-            if (this.path.endsWith(".zarr")) {
+            if (this.path.includes(".zarr")) {
```

Comment:
What situation does this improve the codebase? I have seen file paths with a trailing slash, but I dont believe those are readable by the dependent package anyway without some clipping.

Reply:
ah I see """Quite a lot of zarr data (e.g. generated by bioformats2raw) has the zarr image at path/to/image.zarr/0 so the path doesn't end with .zarr. Hopefully not too many non-zarr images have .zarr in the path, so it shouldn't get many false positives.""" 

I think currently we expect users to provide the top level zarr path. I haven't considered reading in portions of the Zarr, `path/to/image.zarr/0` for example

Reply:
>Quite a lot of zarr data (e.g. generated by bioformats2raw) has the zarr image at path/to/image.zarr/0
Hmm this is good to know - unfortunate from a consistency standpoint, but this seems fine to include for now